src/Microsoft.Azure.WebJobs.Host/Bindings/BindingContext.cs

```diff
@@ -10,7 +10,10 @@
 
 namespace Microsoft.Azure.WebJobs.Host.Bindings
 {
-    internal class BindingContext
+    /// <summary>
+    /// Context for parameter binding.
+    /// </summary>
+    public class BindingContext
```

Comment:
For all these BindingContext types that have been made public, before release I need to make a pass over all their public surface area and reduce what I can. This is an open TODO.